docs/installation.rst

```diff
@@ -162,6 +165,10 @@ Here's the list of the subpackages and what they enable:
 |                     |                                                     | example, ``mysqlclient`` 1.3.12 can only be                          |
 |                     |                                                     | used with MySQL server 5.6.4 through 5.7.                            |
 +---------------------+-----------------------------------------------------+----------------------------------------------------------------------+
+| odbc                | ``pip install 'apache-airflow[odbc]'``              | ODBC data sources including MS SQL Server.  Can use MsSqlOperator,   |
+|                     |                                                     | or as metastore database backend.  Uses pyodbc.                      |
+|                     |                                                     | See :ref:`howto/connection/odbc` for more info.                      |
++---------------------+-----------------------------------------------------+----------------------------------------------------------------------+
```

Comment:
Seems a little strange to duplicate the `apache-airflow[odbc]` package.

Reply:
ok you're right i was thinking this table was about  datasources, but it's about subpackages... i'll update